### PR TITLE
update metadata for A4Education

### PR DESCRIPTION
**Pull request by vinnieangel**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 1]
```diff
@@ -10,14 +10,37 @@
...
-    "isActive": "true"
+    "isA4EActive": "true",
+    "usefulLink": [
+            {
+                "name": "Getting Started",
+                "rank": 1,
+                "url": "https://azure.microsoft.com/en-us/resources/templates/logic-app-create/"
+            },
+            {
+                "name": "Azure Logic App",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:30]: Official product name is "Azure Logic Apps":

```suggestion
                "name": "Azure Logic Apps",
```

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 2]
```diff
@@ -10,14 +10,37 @@
   "type": "QuickStart",  
   "tags": {
     "subscriptiontype": "Azure4Student",
+    "id": 21,
+    "rank": 7,
     "resource": "Logic Apps, MySQL",
     "os": "Linux, Logic App",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:16]:
```suggestion
    "os": "Linux, Azure Logic Apps",
```

**Review by ecfan — CHANGES_REQUESTED** [review]
Added feedback, thanks!

**Review by ecfan — APPROVED** [review]
Thank you!

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 3]
```diff
@@ -1,41 +1,46 @@
...
-    "roles": "Developer",
-    "learning": "https://docs.microsoft.com/en-us/learn/browse/?products=azure-logic-apps&WT.mc_id=Portal-Microsoft_Azure_Education",
-    "isA4EActive": "true",
-    "usefulLink": [
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "icon": "logic",
+    "itemDisplayName": "Create a Consumption logic app using an Azure Resource Manager template",
+    "description": "This template creates an empty logic app that can be used to define workflows.",
+    "summary": "Create a Consumption logic app with an ARM template.",
+    "githubUsername": "jeffhollan",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:7]: Let's update the owner as Jeff has moved to bigger and better challenges. :)

```suggestion
    "githubUsername": "ecfan",
```

**Review by ecfan — CHANGES_REQUESTED** [review]
A few more edits, thanks!

**Comment by ecfan** [comment]
@vinnieangel: Please make the suggested changes to get your PR merged. Otherwise, the system will close the PR after 7 days. Thanks!

**Comment by ecfan** [comment]
#sign-off

**Comment by ecfan** [comment]
#sign-off

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 4]
```diff
@@ -1,41 +1,46 @@
...
+        "cost": "$",
+        "scenario": "Azure Logic Apps",
+        "roles": "Developer",
+        "learning": "https://docs.microsoft.com/en-us/learn/browse/?products=azure-logic-apps&WT.mc_id=Portal-Microsoft_Azure_Education",
+        "isA4EActive": "true",
+        "usefulLink": [
             {
                 "name": "Getting Started",
                 "rank": 1,
                 "url": "https://azure.microsoft.com/en-us/resources/templates/logic-app-create/"
```
Comment (bmoore-msft) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:27]: acom links are being deprecated

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request closed without merging**